### DISCUSSION TITLE
How to deal with "&" for shortcuts in translatable strings?

### DISCUSSION THREAD

**Discussion by buhtz** [WeblateOrg/weblate#9585] [Q&A] [answered]
I do use GNU gettext to translate strings in my Python Qt GUI.

I do have menu entries like this: `_('&Open')`
In German this would be translated into `'&Öffnen` and also change the shortcut from `O` to `Ö`.

Another example is `_('&General')`.
Translated into Russian: `'&Общие'` result in an `O` shortcut which maybe override another `O` shortcut.

How do you control things like this in your applications? Should I avoid using the `&` in such strings? Or is there some GNU gettext or Weblate magic feature to handle this?

**Comment by nijel** [comment]
There is currently no support for this in Weblate, gettext has some, see https://github.com/WeblateOrg/weblate/issues/5962

**Reply by buhtz** [reply, marked as answer]
The `msgfmt --check-accelerators` does not help. It checks only the presence of such a character and not of its uniquness or conflicts in other strings.
